CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
   * IDs generated for entities in the Edge no longer use underscores and instead use periods to avoid issues when used as a common name in CSRs
   * [edge#424](https://github.com/openziti/edge/issues/424) Authenticated, non-admin, clients can query service terminators
   * [sdk-golang#112](https://github.com/openziti/sdk-golang/issues/112) Process checks for Windows are case-insensitive
+  * The CLI agent now runs over unix sockets and is enabled by default. See doc/ops-agent.md for details in the ziti repository.
```

Comment:
So this won't work for windows?

Reply:
According to my research, unix sockets are supported on Win 10 (though I haven't tested it). If it doesn't work, or you have an older version of windows, you still have the option of using tcp sockets instead.

Reply:
https://devblogs.microsoft.com/commandline/af_unix-comes-to-windows/